extension/css/cryptup.css

```diff
@@ -1,4 +1,8 @@
 /* © 2016-2018 FlowCrypt Limited. Limitations apply. Contact human@flowcrypt.com */
+.height-0 {
+  height: 0;
+}
+
```

Comment:
looks like it's not used anywhere, can be removed

Reply:
yes, can remove now.